dask/dataframe/groupby.py

```diff
@@ -1023,6 +1023,12 @@ def _cumcount_aggregate(a, b, fill_value=None):
     return a.add(b, fill_value=fill_value) + 1
 
 
+def _fillna_group(group, by, value, method, limit, fillna_axis):
+    return group.drop(columns=by).fillna(
```

Comment:
I think a comment as to why we are dropping columns would be useful, since it's not super obvious to the reader (namely, that `apply` keeps the grouped-by columns around)

Reply:
Sounds good, added!